python/ray/tune/manager.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import requests
+import json
+import threading
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from six.moves.queue import Queue
+
+import ray
+from ray.tune.error import TuneError, TuneManagerError
+from ray.tune.variant_generator import generate_trials
+
+@ray.remote
+def _send_request(path, payload):
```

Comment:
Does this really need to be `@ray.remote`? It would make sense for this class to not depend on ray.